docs/dev/guide/l1-l2.md

```diff
@@ -63,12 +65,11 @@ function requestL2Transaction(
 
 - `_contractAddressL2` is a parameter that defines the address of the contract to be called.
 - `_calldata` is a parameter that contains the calldata of the transaction call. It can be encoded the same way as on Ethereum.
-- `_ergsLimit` is a parameter that contains the `ergs` limit of the transaction call. You can learn more about `ergs` and the zkSync fee system [here](../zksync-v2/fee-model).
-- `_factoryDeps` is a list of bytecodes. It should contain the bytecode of the contract being deployed. If the contract being deployed is a factory contract, i.e. it can deploy other contracts, the array should also contain the bytecodes of the contracts which can be deployed by it.
+- `_ergsLimit` is a parameter that contains the ergs limit of the transaction call. You can learn more about ergs and the zkSync fee system [here](../zksync-v2/fee-model).
+- `_factoryDeps` is a list of bytecodes. It should contain the bytecode of the contract being deployed. If the contract being deployed is a factory contract, i.e. it can deploy other contracts, the array should also contain the bytecodes of the contracts that can be deployed by it.
 - `_queueType` is a parameter required for the priority mode functionality. For the testnet, `QueueType.Deque` should always be supplied.
 
-With the method call, some amount of ETH should be supplied to cover the base cost of the transaction + layer 2 operator tip. The function returns canonical hash or requested transaction, that can be used to track the execution of the transaction in L2.
-
+With the method call, some amount of ETH should be supplied to cover the base cost of the transaction + layer 2 operator tip.
```

Comment:
I think it is better to have an empty line before the beginning of the next section

```suggestion
With the method call, some amount of ETH should be supplied to cover the base cost of the transaction + layer 2 operator tip.

```